dockerfiles/gwt-ide/traefik.toml

```diff
@@ -0,0 +1,74 @@
+################################################################
+# Global configuration
+################################################################
+
+debug = true
+logLevel = "DEBUG"
+[traefikLog]
+  filePath = "/home/user/agent/traefik/traefik.log"
+[accessLog]
+  filePath = "/home/user/agent/traefik/access.log"
+
+
+# Entrypoints to be used by frontends that do not specify any entrypoint.
+# Each frontend can specify its own entrypoints.
+#
+# Optional
+# Default: ["http"]
+#
+defaultEntryPoints = ["http"]
+
+# Entrypoints definition
+#
+# Optional
+# Default:
+[entryPoints]
+    [entryPoints.http]
+    address = ":8085"
+    [entryPoints.http2]
+    address = ":8087"
+
+################################################################
+# Web configuration backend
+################################################################
+
+# Enable web configuration backend
+#[web]
+
+# Web administration port
+#
+# Required
+#
+#address = ":8086"
+
+
+
+[file]
+
+[backends]
+  [backends.backend1]
```

Comment:
Please consider using meaningful names for backends, like `apiBackend`, `ideBackend`